src/pybind/mgr/volumes/fs/async_job.py

```diff
@@ -105,6 +104,9 @@ class AsyncJobs(threading.Thread):
     via `should_cancel()` lambda passed to `execute_job()`.
     """
 
+    # not made configurable on purpose
+    WAKEUP_TIMEOUT = 5.0
```

Comment:
just curious to know why we didn't go with the option to make it configurable?

Reply:
since it hardly matters if a "manual" work is picked up now or 5s later.